helix-core/src/test/java/org/apache/helix/model/TestClusterConfig.java

```diff
@@ -320,6 +320,29 @@ public void testSetOfflineNodeTimeOutForMaintenanceMode() {
             -1), 10000L);
   }
 
+
+  @Test
+  public void testGetOfflineNodeTimeOutForPurge() {
+    ClusterConfig testConfig = new ClusterConfig("testId");
+    Assert.assertEquals(testConfig.getOfflineNodeTimeOutForPurge(), -1);
+
+    testConfig.getRecord()
+        .setLongField(ClusterConfig.ClusterConfigProperty.OFFLINE_NODE_TIME_OUT_FOR_PURGE
+                .name(),
+            10000L);
+    Assert.assertEquals(testConfig.getOfflineNodeTimeOutForPurge(), 10000L);
+  }
+
+  @Test
+  public void testSetOfflineNodeTimeOutForPurge() {
```

Comment:
This test really is not necessary. All the logic seems tested in testGetOfflineNodeTimeOutForPurge() anyway.

Reply:
Tiny difference in how they set the field. But I agree they're doing the same thing.